plugins/woocommerce/includes/admin/helper/class-wc-helper.php

```diff
@@ -681,6 +681,30 @@ public static function current_screen( $screen ) {
 		}
 	}
 
+	/**
+	 * Get helper redirect URL.
+	 * @param array $args Query args.
+	 * @return string
+	 */
+	private static function _get_helper_redirect_url( $args = array() ) {
```

Comment:
Can we do away with the `_` prefix? It's already marked as `private` so nobody can call this anyway. 

```suggestion
	private static function get_helper_redirect_url( $args = array() ) {
```